SingularityUI/app/handlebarsHelpers.coffee

```diff
@@ -119,3 +119,10 @@ Handlebars.registerHelper 'getLabelClass', (state) ->
             'danger'
         else
             'default'
+
+Handlebars.registerHelper 'isRunningState', (list, options) ->
+    switch _.last(list).taskState
```

Comment:
use `@` for `this`. my personal (some others too) opinion, but if we're just testing one condition, I find if statements easier to read and just as fast